R/teal_data-get_code.R

```diff
@@ -0,0 +1,50 @@
+#' Get Code from `teal_data`
+#'
+#' Get code from [`teal_data`]. The extraction of the code limited to specific objects can be performed with the usage
+#' of `names` parameter. If input `teal_data@verified` is `FALSE` the returned code is pre-pended with a warning
+#' stating that the object was not verified.
+#'
+#' @param x [`teal_data`] object.
+#' @param names (`character`) vector of object names to return the code for.
+#' @param deparse (`logical`) whether return deparsed form of a call.
+#' @return (`character`) code of import and preparation of data for `teal` application.
+#' @examples
+#'
+#' tdata1 <- teal_data()
+#' tdata1 <- within(tdata1, {
+#'   a <- 1
+#'   b <- a^5
+#'   c <- list(x = 2)
+#' })
+#' get_code(tdata1)
+#' datanames(tdata1) <- c("a", "b", "c")
+#' get_code(tdata1, "a")
+#' get_code(tdata1, "b")
+#'
+#' tdata2 <- teal_data(x1 = iris, code = "x1 = iris")
+#' get_code(tdata2)
+#' get_code(verify(tdata2))
+#' @aliases get_code,teal_data-method
```

Comment:
```suggestion
#' 
#' @name get_code
#' @rdname get_code
#' @aliases get_code,teal_data-method
#' 
#' @seealso [`teal_data`], [`datanames`], [`verify`]
#' 
```

Reply:
Unsure about 

```
#' @name get_code
#' @rdname get_code
```

as we still have those in `R/get_code.R` for `get_code.TealDatasetConnector` , `get_code.TealDataset` and `get_code.TealDataAbstract`.

I think I would add to `R/teal_data-get_code.R`
```
#' @name get_code
#' @rdname get_code
```
on a branch where we remove `R/get_code.R` for `get_code.TealDatasetConnector` , `get_code.TealDataset` and `get_code.TealDataAbstract`.

Reply:
Right. I assumed no further conflicts would arise. Please test and if they do, postpone these changes.

Reply:
yeah. tested this a couple of times and this messes documentation pages from 2 files. will make a card to incorporate that, once get_code.TealDatasetConnector , get_code.TealDataset and get_code.TealDataAbstract are removed

Reply:
https://github.com/insightsengineering/teal.data/issues/173#issuecomment-1822603906